runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -651,12 +651,33 @@ class CompilationInfo
 #if defined(J9VM_OPT_JITSERVER)
       TR_ASSERT_FATAL(!TR::CompilationInfo::getStream(), "not yet implemented for JITServer");
 #endif /* defined(J9VM_OPT_JITSERVER) */
-      value = (value << 1) | 1;
-      if (value < 0)
-          value = INT_MAX;
-      method->extra = reinterpret_cast<void *>(value);
+#if defined(J9VM_OPT_MICROJIT)
+      TR::Options *options = TR::Options::getJITCmdLineOptions();
+      if(options->_mjitEnabled && value)
+         {
+            return;
+         }
+         else
+#endif
+         {
+         value = (value << 1) | 1;
+         if (value < 0)
+            value = INT_MAX;
+         method->extra = reinterpret_cast<void *>(value);
+         }
       }
 
+#if defined(J9VM_OPT_MICROJIT)
+   static void setInitialMJITCountUnsynchronized(J9Method *method, int32_t mjitThreshold, int32_t trThreshold)
```

Comment:
`trThreshold` is not used in this function. Is there a plan to use it in the future? If not, this parameter should be removed.

Reply:
There was an idea to possibly use this (or some function over the set of numbers supplied to the parameter, since it's a per-method function call) to inform MicroJIT about what to use as a threshold for recompiling with TR. I do not remember offhand if that was tabled or not.